GitUI/CommandsDialogs/SettingsDialog/Pages/GitConfigSettingsPage.Designer.cs

```diff
@@ -170,6 +170,7 @@ private void InitializeComponent()
             this.txtDiffToolPath.Location = new System.Drawing.Point(129, 200);
             this.txtDiffToolPath.Name = "txtDiffToolPath";
             this.txtDiffToolPath.Size = new System.Drawing.Size(902, 20);
+            this.txtDiffToolPath.TextChanged += new System.EventHandler(this.txtDiffToolPath_TextChanged);
             this.txtDiffToolPath.LostFocus += new System.EventHandler(this.txtDiffMergeToolPath_LostFocus);
```

Comment:
nit, the designer orders statements in A-Z order
```suggestion
            this.txtDiffToolPath.LostFocus += new System.EventHandler(this.txtDiffMergeToolPath_LostFocus);
            this.txtDiffToolPath.TextChanged += new System.EventHandler(this.txtDiffToolPath_TextChanged);
```

Reply:
You are right - strange enough I didn't add it manually. Reordered.